pkg/logql/bench/bench_test.go

```diff
@@ -100,18 +119,24 @@ func TestStorageEquality(t *testing.T) {
 		}
 	}
 
-	// Generate a list of stores. The first store name provided here is the one
-	// that acts as the baseline.
-	var stores []*store
-	for _, name := range []string{"chunk", "dataobj"} {
+	// Generate a list of stores. The chunks store (as the most stable) acts as
+	// the baseline.
+	var (
+		stores    []*store
+		baseStore *store
+	)
+	for _, name := range allStores {
 		store := generateStore(name)
 		stores = append(stores, store)
+
+		if name == StoreChunk {
+			baseStore = store
+		}
 	}
 	if len(stores) < 2 {
 		t.Skipf("not enough stores to compare; need at least 2, got %d", len(stores))
 	}
 
-	baseStore := stores[0]
```

Comment:
The loop below is wrong now, it's assuming stores[0] is the base store (I can fix this part)